test/main.js

```diff
@@ -2,7 +2,7 @@ const should = require('should');
 const Vinyl = require('vinyl');
 const path = require('path');
 const fs = require('fs');
-const sass = require('../index');
+const sass = require('../index')(require('node-sass'));
```

Comment:
🔍 ⚠️ 

I tried to use the Dart Sass compiler in the tests, and I can't get them to pass. I used this version of the code, including the Dart Sass compiler, on one of my projects, and it worked fine. I think the problem is not with the actual software implementation, but with how Dart Sass interacts with the tests themselves. Unfortunately, I do not know enough about Node or Mocha to guess what a better test might look like. I just wanted to highlight that in case high confidence about Dart Sass is important before we ship.

Reply:
Let's not get hung up on the getting tests passing with dart sass. if you can get it working locally then it can be dealt with in AA patch